src/linux/ldcache.cpp

```diff
@@ -224,7 +224,7 @@ Try<vector<Entry>> parse(const string& path)
   // contained in the string table. At this point, 'data' should
   // point to an address just beyond the end of the file.
```

Comment:
I think the comment is stale, since `data` won't necessarily point just beyond the end of the file anymore.

Similarly, the comment below will also be stale:
https://github.com/apache/mesos/blob/15c493c4dba2bf7b5db514e1daefc9c8d4f11bf6/src/linux/ldcache.cpp#L231

```
  // Make sure the very last character in the buffer is a '\0'.
```